PublicFolders/ValidateEXOPFDumpster.ps1

```diff
@@ -364,7 +400,35 @@ function ExtractLog {
     if (!(Test-Path  "$ExportPath\logs_$ts")) {
         mkdir "$ExportPath\logs_$ts" -Force | Out-Null
     }
-    $PublicFolderInfo | Export-Clixml -Path "$ExportPath\logs_$ts\PublicFolderInfo.xml"
+    $PublicFolderInfo | Export-Clixml -Path "$ExportPath\logs_$ts\PublicFolderInfo$($PublicFolderInfo.PublicFolder.Name).xml"
+}
+function AskForFeedback {
+    Write-Host "Please rate the script experience & tell us what you liked or what we can do better over https://aka.ms/PFDumpsterFeedback!" -ForegroundColor Cyan
+    "Please rate the script experience & tell us what you liked or what we can do better over https://aka.ms/PFDumpsterFeedback!" | Out-File $ExportPath\$Script:ReportName -Append
+}
+function QuitEXOSession {
+    if ($null -eq $SessionCheck) {
+        try {
+            Write-Host "Quitting EXO PowerShell session..." -ForegroundColor Yellow
+            Disconnect-ExchangeOnline -ErrorAction Stop -Confirm:$false
+            $CurrentDescription= "Disconnecting from EXO"
+            $CurrentStatus = "Success"
+            LogError -CurrentStatus $CurrentStatus -Function "Disconnecting from EXO" -CurrentDescription $CurrentDescription
+            Write-Host "`nOutput was exported in the following location: $ExportPath" -ForegroundColor Yellow
+            Start-Sleep -Seconds 3
+            break
+        } catch {
+            $ErrorEncountered=$Global:error[0].Exception
+            $CurrentDescription = "Disconnecting from EXO"
+            $CurrentStatus = "Failure"
+            LogError -CurrentStatus $CurrentStatus -Function "Disconnecting from EXO" -CurrentDescription $CurrentDescription
+            Write-Host "Error encountered during executing the script!"-ForegroundColor Red
+            Write-Host $ErrorEncountered -ForegroundColor Red
+            Write-Host "`nOutput was exported in the following location: $ExportPath" -ForegroundColor Yellow
+            Start-Sleep -Seconds 3
+            break
```

Comment:
Not sure what the intent of this `break` is here, but this isn't correct use of it. 

https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_break?view=powershell-7.3#do-not-use-break-outside-of-a-loop-switch-or-trap

Reply:
Removed.